packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -42,32 +43,40 @@ const { details: X } = assert;
  * collateralTypes: Store<Brand,VaultManager>,
  * directorParamManager: import('@agoric/governance/src/contractGovernance/typedParamManager').TypedParamManager<import('./params.js').VaultDirectorParams>,
  * metricsPublication: IterationObserver<MetricsNotification>
- * metricsSubscription: Subscription<MetricsNotification>
+ * metricsSubscription: StoredSubscription<MetricsNotification>
```

Comment:
Please rename `subscription` to `subscriber` before it becomes too entrenched.  Sorry, I should have specified this more clearly in my last review.

```suggestion
 * metricsSubscriber: StoredSubscriber<MetricsNotification>
```

Reply:
How can it become "too entrenched"? How would it be any less work to do rename now that in a later PR?

Reply:
Oh, I shouldn't have recommended changing  `metricsSubscription` since it was already there.  In this case, the only useful part of my comment is changing `StoredSubscription` to `StoredSubscriber`.